engines/ags/tests/test_gfx.cpp

```diff
@@ -19,28 +19,257 @@
+											uint32 controlCol, simdCol, pixelDummy;
+											uint16 control2bppCol, simd2bppCol, pixelDummy2bpp;
+											uint8 a = destA, r = destR, g = destG, b = destB;
+											pixelDummy = (a << 24) | (r << 16) | (g << 8) | b;
+											dummy.blendPixel(srcA, srcR, srcG, srcB, a, r, g, b, alpha, false, (byte *)&pixelDummy);
+											controlCol = b | (g << 8) | (r << 16) | (a << 24);
+
+											uint8 a16 = 0xff, r16 = destR >> 3, g16 = destG >> 2, b16 = destB >> 3;
+											r16 = (r16 << 3) | (r16 >> 2);
+											g16 = (g16 << 2) | (g16 >> 4);
+											b16 = (b16 << 3) | (b16 >> 2);
+											uint8 srcR16 = srcR >> 3, srcG16 = srcG >> 2, srcB16 = srcB >> 3;
+											srcR16 = (srcR16 << 3) | (srcR16 >> 2);
+											srcG16 = (srcG16 << 2) | (srcG16 >> 4);
+											srcB16 = (srcB16 << 3) | (srcB16 >> 2);
+											pixelDummy2bpp = (destB >> 3) | ((destG >> 2) << 5) | ((destR >> 3) << 11);
+											dummy16.blendPixel(0xff, srcR16, srcG16, srcB16, a16, r16, g16, b16, alpha, false, (byte *)&pixelDummy2bpp);
+											r16 >>= 3; g16 >>= 2; b16 >>= 3;
+											control2bppCol = b16 | (g16 << 5) | (r16 << 11);
+											{
+#ifdef OPT_NEON
+												uint32x4_t src = vdupq_n_u32(srcB | (srcG << 8) | (srcR << 16) | (srcA << 24));
+												uint32x4_t dest = vdupq_n_u32(destB | (destG << 8) | (destR << 16) | (destA << 24));
+												uint32x4_t alphas = vdupq_n_u32(alpha);
+												simdCol = vgetq_lane_u32(blendPixelSIMD(src, dest, alphas), 0);
+#else
+												__m128i src = _mm_set1_epi32(srcB | (srcG << 8) | (srcR << 16) | (srcA << 24));
+												__m128i dest = _mm_set1_epi32(destB | (destG << 8) | (destR << 16) | (destA << 24));
+												__m128i alphas = _mm_set1_epi32((int)alpha);
+												simdCol = _mm_cvtsi128_si32(blendPixelSIMD(src, dest, alphas));
+#endif
+											}
+											{
+#ifdef OPT_NEON
+												uint16x8_t src = vdupq_n_u16((srcB >> 3) | ((srcG >> 2) << 5) | ((srcR >> 3) << 11));
+												uint16x8_t dest = vdupq_n_u16((destB >> 3) | ((destG >> 2) << 5) | ((destR >> 3) << 11));
+												uint16x8_t alphas = vdupq_n_u16((uint16)alpha);
+												simd2bppCol = vgetq_lane_u16(blendPixelSIMD2Bpp(src, dest, alphas), 0);
+#else
+												__m128i src = _mm_set1_epi16((srcB >> 3) | ((srcG >> 2) << 5) | ((srcR >> 3) << 11));
+												__m128i dest = _mm_set1_epi16((destB >> 3) | ((destG >> 2) << 5) | ((destR >> 3) << 11));
+												__m128i alphas = _mm_set1_epi16((uint16)alpha);
+												simd2bppCol = (uint16)(_mm_cvtsi128_si32(blendPixelSIMD2Bpp(src, dest, alphas)) & 0xffff);
+#endif
+											}
+#ifdef VERBOSE_TEST_GFX
+											debug("src argb: %d, %d, %d, %d dest argb: %d, %d, %d, %d a: %d", srcA, srcR, srcG, srcB, destA, destR, destG, destB, alpha);
+#endif
+											switch ((BlenderMode)blenderMode) {
+												case kSourceAlphaBlender: debug("blenderMode: kSourceAlphaBlender"); break;
```

Comment:
... we indent switch() statement differently. Please check our Code Formatting Conventions